src/screens/Login/Login.js

```diff
@@ -33,9 +38,9 @@ const Login = props => {
           navigation.navigate("mainPage");
         }
       })
-      .catch(() => {
-        // TODO: show message for user
-        Alert.alert(`Sorry, there's a problem with your data`);
+      .catch(err => {
+        setError(err.message || translate("login.backendErrorMessage"));
```

Comment:
Should we show `translate("login.backendErrorMessage")` only? Do we have the correct message from backend?

Reply:
When we have 403 error from backend so we will show the text for this error. If other error we will get the message from backend and show it. Finally, when we haven't any error message we will show placeholder message to avoid empty modal window and notify user that something goes wrong

Reply:
Okay, but what kind of errors do we have from backend? As I know it should be only in English, what is not correct for mobile users.

Reply:
403 error is translated using our translate service. I added possibility to add another translations if needed. If there are a list of errors from backend we can simply integrate this